vendor/github.com/v3io/v3io-go/pkg/dataplane/streamconsumergroup/statehandler.go

```diff
@@ -180,11 +187,20 @@ func (sh *stateHandler) assignShards(maxReplicas int, numShards int, state *Stat
 	// empty shard groups are not unique - therefore simply check whether the number of
 	// empty shard groups allocated to sessions is equal to the number of empty shard groups
 	// required. if not, allocate an empty shard group
+	sh.logger.DebugWith("Getting assign empty shard group",
```

Comment:
should be leveled and inside the if. we want to see that we hit the condition, not that we're gonna check for it